chart/values.yaml

```
@@ -2505,6 +2505,9 @@ dags:
     # and specify the name of the secret below
     # sshKeySecret: airflow-ssh-secret
     #
+    # Or set sshKeySecret with your key
+    # sshKey: ~
```

```suggestion
    # Or set sshKey with your key
    # sshKey: |-
    #   -----BEGIN OPENSSH PRIVATE KEY-----
    #   ...
    #   -----END OPENSSH PRIVATE KEY-----
```

This is more what I was thinking.

---

seems like this test does not allow it  "Detect if private key is added to the repository"

---

Maybe mangle the text a bit so the test doesn't hit on it, but it's (hopefully) obvious to a human what should be there?

```
-----BEGIN {YOUR-PRIVATE-KEY}-----
...
-----END {YOUR-PRIVATE KEY}-----
```